src/lib/components/Map/Map.js

```diff
@@ -510,8 +540,19 @@ class Map extends Component {
       } catch (e) {
         console.warn('resize error', e);
       }
+      
+      const { layersObj, layerObj, primaryLayer, FILTER, LOC, mapId, timeSeriesObj, APP, VIEW } = this.props;
+
+      const layers = this.props.layers;
+      const CurrPrimaryLayer = this.props.primaryLayer;
+      if (this.props.hasDataView && this.map && VIEW.showLayerSuperset) {
+        if (layers && layers[CurrPrimaryLayer] && layers[CurrPrimaryLayer].location) {
+          this.map.easeTo(layers[CurrPrimaryLayer].location);
+        } else {
+          this.dataViewMapReset(APP.mapConfig, this.map)
+        }
+      }
```

Comment:
@ciremusyoka is there a reason we're defining `CurrPrimaryLayer` instead of just using `primaryLayer`?